scripts/scripts.js

```diff
@@ -12,7 +12,7 @@
 
 const PROJECT_NAME = 'milo--adobecom';
 const PRODUCTION_DOMAINS = ['milo.adobe.com'];
-const LCP_BLOCKS = ['section', 'hero'];
+const LCP_BLOCKS = ['section', 'hero', 'how-to', 'faq'];
```

Comment:
I'd ditch this. I don't think these would ever be in contention for being LCP. We want to keep this array pretty small. In fact, I need to remove section.